app/modules/bookings/book_tickets/book_tickets.validator.js

```diff
@@ -1,9 +1,26 @@
 const {body} = require ('express-validator');
+const {tickets:Tickets} = require('../../../../models');
 
 const bookTicketsvalidator  = [
     
     body('quantity')
-    .isInt(),
+    .isInt()
```

Comment:
You can add custom messages. Please try that out.
Example: .withMessage('Quantity is required.').


Also, try out .custom()
This can be used to write custom validations. Eg: you can conditionally validate the req.